Gemfile.lock

```diff
@@ -104,55 +104,55 @@ GEM
       mini_mime (>= 0.1.1)
     marcel (0.3.3)
       mimemagic (~> 0.3.2)
-    method_source (0.9.2)
+    method_source (1.0.0)
     mimemagic (0.3.4)
     mini_mime (1.0.2)
     mini_portile2 (2.4.0)
     minitest (5.14.0)
     msgpack (1.3.3)
     nio4r (2.5.2)
-    nokogiri (1.10.8)
+    nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     popper_js (1.16.0)
     public_suffix (4.0.3)
-    puma (4.3.1)
+    puma (4.3.3)
       nio4r (~> 2.0)
     rack (2.2.2)
     rack-test (1.1.0)
       rack (>= 1.0, < 3)
-    rails (6.0.2.1)
-      actioncable (= 6.0.2.1)
-      actionmailbox (= 6.0.2.1)
-      actionmailer (= 6.0.2.1)
-      actionpack (= 6.0.2.1)
-      actiontext (= 6.0.2.1)
-      actionview (= 6.0.2.1)
-      activejob (= 6.0.2.1)
-      activemodel (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
-      activestorage (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    rails (6.0.2.2)
+      actioncable (= 6.0.2.2)
+      actionmailbox (= 6.0.2.2)
+      actionmailer (= 6.0.2.2)
+      actionpack (= 6.0.2.2)
+      actiontext (= 6.0.2.2)
+      actionview (= 6.0.2.2)
+      activejob (= 6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
+      activestorage (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       bundler (>= 1.3.0)
-      railties (= 6.0.2.1)
+      railties (= 6.0.2.2)
       sprockets-rails (>= 2.0.0)
     rails-dom-testing (2.0.3)
       activesupport (>= 4.2.0)
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    railties (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       method_source
       rake (>= 0.8.7)
       thor (>= 0.20.3, < 2.0)
     rake (13.0.1)
     rb-fsevent (0.10.3)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    regexp_parser (1.6.0)
+    regexp_parser (1.7.0)
     ruby_dep (1.5.0)
-    rubyzip (2.2.0)
+    rubyzip (2.3.0)
```

Comment:
### [rubyzip](http://github.com/rubyzip/rubyzip)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.0 → 2.3.0

[_[change-log](https://github.com/rubyzip/rubyzip/blob/v2.3.0/Changelog.md), [source-code](https://github.com/rubyzip/rubyzip/tree/v2.3.0)_]





<details>
<summary>Commits</summary>

A change of **85** commits. See the full changes on [the compare page](https://github.com/rubyzip/rubyzip/compare/v2.2.0...v2.3.0).

These are the first 10 commits:
- (609c12d) [added Ruby 2.7 to CI regression](https://github.com/rubyzip/rubyzip/commit/609c12d1c484d04e056dc3e39a47cd45a887f1dd)
- (3bc85cc) [fixed frozen error caseud by frozen string literal](https://github.com/rubyzip/rubyzip/commit/3bc85ccdecfb9be015ecde2227fe9db202c7ea0d)
- (b326d17) [use @+ operator instead of #dup to get unfrozen string](https://github.com/rubyzip/rubyzip/commit/b326d1743890b7429472c06403f5294e18d38c34)
- (976dbd3) [reverted the change according to comment below:](https://github.com/rubyzip/rubyzip/commit/976dbd34081af9b0c8c1970410e787d32fb7cc5c)
- (393cd97) [Merge pull request #431 from taichi-ishitani/fix_frozen_erro…](https://github.com/rubyzip/rubyzip/commit/393cd9749c798f61e99064bdd851eff199eaf468)
- (247fd43) [Update changelog for #431](https://github.com/rubyzip/rubyzip/commit/247fd432901b143ed37c11a90da3c6c0f01489b8)
- (862892c) [Quickly fire up a console with 'zip' pre-loaded.](https://github.com/rubyzip/rubyzip/commit/862892c0ac47af43ed8df6f9dd3f6b751dfbdf38)
- (5da87ef) [Merge pull request #420 from hainesr/console](https://github.com/rubyzip/rubyzip/commit/5da87ef1d6043d9f428de31c692ffba416563826)
- (c8bfd14) [Update rubocop version and the config files.](https://github.com/rubyzip/rubyzip/commit/c8bfd147643cf783c779ec6c770077c8394353a5)
- (a3245ac) [Add Rubocop tasks to the Rakefile.](https://github.com/rubyzip/rubyzip/commit/a3245ac241e9a33bacdffbe1e45f79344b46f23e)


</details>